app/display/linearmeter/src/main/java/org/csstudio/display/extra/widgets/linearmeter/LinearMeterRepresentation.java

```diff
@@ -336,12 +338,15 @@ private void valueChanged(WidgetProperty<?> property, Object old_value, Object n
                                     && controlRange.getMaximum() - controlRange.getMinimum() > 0.0) {
                                 if (meter.linearMeterScale.getValueRange().getLow() != controlRange.getMinimum() || meter.linearMeterScale.getValueRange().getHigh() != controlRange.getMaximum() || !meter.getValidRange()) {
                                     meter.setRange(controlRange.getMinimum(), controlRange.getMaximum(), true);
+                                    linearMeterScaleHasChanged = true;
```

Comment:
This makes me feel like this should be a property of the linearmeter, so this should be something like:

meter.setScaleUpdated()